Makefile

```diff
@@ -90,6 +100,8 @@ manifests: controller-gen kustomize ## Generate WebhookConfiguration, ClusterRol
 # generate the <package-name>.clusterserviceversion.yaml
 config/manifests/bases/$(PACKAGE_NAME).clusterserviceversion.yaml: config/manifests/bases/clusterserviceversion.yaml.in
 	sed 's/@PACKAGE_NAME@/$(PACKAGE_NAME)/g' < $^ > $@
+	sed -i 's/@SKIP_RANGE@/$(SKIP_RANGE)/g' $@
```

Comment:
`-i` is required?

Reply:
Yes, that tells sed to make the replace inplace